### PR TITLE
Feat/guided answers/header visibility 90

### PR DESCRIPTION
**Pull request by unseen1980**
This PR addresses the tasks from this story https://github.com/SAP/guided-answers-extension/issues/90

1. Changes in folder structure for easier navigation
2. Move the header section into a new component and add props for the logo and nav button
3. Toggle off part of the header in the Details section

Screenshot 2022-06-08 at 09 34 35

Split Header into smaller components
Screenshot 2022-06-08 at 09 58 59

### CODE REVIEW

File: .changeset/long-spoons-pump.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@sap/guided-answers-extension-types': minor
```
Comment (Klaus-Keller) [inline .changeset/long-spoons-pump.md:2]: @unseen1980, in order to generate a new version of the extension itself, we need also to create a changeset for `sap-guided-answers-extension`, even if it is not changed, the version needs to be bumped.

**Review by Klaus-Keller — COMMENTED** [review]
@unseen1980, please see my comments.

File: packages/webapp/src/webview/ui/components/App/App.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * @returns - react elements for this app
+ */
+export function App(): ReactElement {
+    const appState = useSelector<AppState, AppState>((state) => state);
+
+    let content;
+    if (appState.activeGuidedAnswerNode.length > 0) {
+        content = <GuidedAnswerNode />;
+    } else if (appState.guidedAnswerTrees) {
+        content = content =
```
[the remark this answers is not in the document]
Comment (unseen1980) [reply packages/webapp/src/webview/ui/components/App/App.tsx]: Changed thanks!

**Review by Klaus-Keller — APPROVED** [review]
- [x] Code looks good
- [x] Build locally and tested

Approved

**Comment by unseen1980** [comment]
Thanks @Klaus-Keller

**Pull request merged**